datasets/ge/company_regisrty/ge_company_registry.yml

```diff
@@ -0,0 +1,254 @@
+title: Georgian Company Registry
+entry_point: crawler.py
+prefix: ge-company-registry
+coverage:
+  frequency: weekly
+  start: 2025-01-20
+load_db_uri: ${OPENSANCTIONS_DATABASE_URI}
```

Comment:
we don't normally load the full company registers into the statements table - have you discussed whether we want to for this one?